lib/Doctrine/ODM/PHPCR/DocumentClassMapper.php

```diff
@@ -50,8 +52,8 @@ public function getClassName(DocumentManagerInterface $dm, NodeInterface $node,
     {
         $className = $this->expandClassName($dm, $className);
 
-        if ($node->hasProperty('phpcr:class')) {
-            $nodeClassName = $node->getProperty('phpcr:class')->getString();
+        if ($node->hasProperty(self::CLASS_PROPERTY)) {
+            $nodeClassName = $node->getProperty(self::CLASS_PROPERTY)->getString();
```

Comment:
Actually this is too risky (adding class constants to places that might be untested) and unrequired for this PR, reverting.